ding/policy/bc.py

```diff
@@ -0,0 +1,196 @@
+import math
+import torch
+import torch.nn as nn
+import copy
+from torch.optim import Adam
+from torch.optim.lr_scheduler import LambdaLR
+from typing import List, Dict, Any, Tuple, Union, Optional
+from collections import namedtuple, deque
+from easydict import EasyDict
+from ding.policy import Policy
+from ding.model import model_wrap
+from ding.torch_utils import to_device
+from ding.utils import EasyTimer
+from ding.utils.data import default_collate, default_decollate
+from ding.rl_utils import q_nstep_td_data, q_nstep_sql_td_error, get_nstep_return_data, get_train_sample
+from ding.worker.collector.interaction_serial_evaluator import InteractionSerialEvaluator
+from ding.utils import POLICY_REGISTRY
+
+
```

Comment:
merge some functions of `policy/il.py` info this `policy/bc.py`, you can ignore some parts code related to gfootball. As the result, the modified code can pass unittest in `ding/entry/tests/test_serial_pipeline_il.py`.